action.yml

```diff
@@ -16,4 +21,4 @@ runs:
         fetch-depth: 2
     - id: is-release
       shell: bash
-      run: ${{ github.action_path }}/scripts/is-release.sh ${{ github.event.before }}
+      run: ${{ github.action_path }}/scripts/is-release.sh "${{ github.event.before }}" "${{ inputs.commit-starts-with }}"
```

Comment:
Had to push a new commit to quote these arguments. Had a string splitting issue with the commit prefix.